themes/bootstrap3/js/lightbox.js

```diff
@@ -467,6 +480,28 @@ VuFind.register('lightbox', function Lightbox() {
       });
     });
   }
+
+  function loadConfiguredLightbox(){
+    if (VuFind.lightbox.child) {
+      // remove lightbox reference
+      let lightboxChild = VuFind.lightbox.child;
+      VuFind.lightbox.child = null;
+      let url = new URL(window.location.href);
+      let params = url.searchParams;
+      params.delete('lightboxChild');
+      url.search = params.toString();
```

Comment:
This can be simplified a bit (and use `const` since the object reference won't change):
```
      const url = new URL(window.location.href);
      url.searchParams.delete('lightboxChild');
```

Reply:
done